cdc/cdc/model/changefeed_test.go

```diff
@@ -119,45 +68,12 @@ func TestFillV1(t *testing.T) {
 		Engine:  "memory",
 		SortDir: ".",
 		Config: &config.ReplicaConfig{
-			CaseSensitive: true,
-			Filter: &config.FilterConfig{
-				MySQLReplicationRules: &filter.MySQLReplicationRules{
-					DoTables: []*filter.Table{{
-						Schema: "test",
-						Name:   "tbl1",
-					}, {
-						Schema: "test",
-						Name:   "tbl2",
-					}},
-					DoDBs: []string{"test1", "sys1"},
-					IgnoreTables: []*filter.Table{{
-						Schema: "test",
-						Name:   "tbl3",
-					}, {
-						Schema: "test",
-						Name:   "tbl4",
-					}},
-					IgnoreDBs: []string{"test", "sys"},
-				},
-				IgnoreTxnStartTs: []uint64{1, 2},
-				DDLAllowlist:     []model.ActionType{1, 2},
-			},
-			Mounter: &config.MounterConfig{
-				WorkerNum: 64,
-			},
 			Sink: &config.SinkConfig{
 				DispatchRules: []*config.DispatchRule{
```

Comment:
Also remove here?

Reply:
`DispatchRules` ? Would be useful when we support message queue as downstream. So I keep them.